File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/analytics/AnalyticsActivity.java
package com.jdroid.android.sample.ui.analytics;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class AnalyticsActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return AnalyticsFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/analytics/AnalyticsActivity.kt
package com.jdroid.android.sample.ui.analytics

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class AnalyticsActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return AnalyticsFragment::class.java
    }
}
